frontend/src/app/components/ExperimentRowCard.js

```diff
@@ -79,6 +86,32 @@ export default class ExperimentRowCard extends React.Component {
     );
   }
 
+  renderFeedbackButton() {
+    if (!this.props.enabled) { return ''; }
```

Comment:
I usually use null to indicate nothing should be rendered, but it's not that important.